# Try Exeption
# try:
#     n = int(input('Enter num: '))

# except ValueError:
#     print('Ошибка')

# except TypeError:
#     print('Value error')
# c = 2 + n
# b = 5
# print(c)

# while True:
#     try:
#         n = int(input('Enter num: '))
#     except Exception as e:
#         print('Error', e)
#     else:
#         print('jslkdjf')
#         print(3+3)
#     finally:
#         n = 33 + 7
#         print(n)

#     b = 5
#     c = 2 + b
#     print(c)

# Decorators

try:
    num = input()

except Exception as e:
    


